cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -339,6 +370,100 @@ impl YaccParser {
         }
     }
 
+    fn parse_yacckind(&mut self, i: usize, benign: bool) -> Result<usize, YaccGrammarError> {
+        const YACC_KINDS: [(&str, YaccKind); 5] = [
+            ("Grmtools", YaccKind::Grmtools),
+            (
+                "Original(NoAction)",
+                YaccKind::Original(YaccOriginalActionKind::NoAction),
+            ),
+            (
+                "Original(UserAction)",
+                YaccKind::Original(YaccOriginalActionKind::UserAction),
+            ),
+            (
+                "Original(GenericParseTree)",
+                YaccKind::Original(YaccOriginalActionKind::GenericParseTree),
+            ),
+            ("Eco", YaccKind::Eco),
```

Comment:
I noticed that nimblepase uses lowercase here, and also has `original` with no action kind specified,
> [-y <eco|grmtools|original>]

Should we try and make this case insensitive?
Not sure about the action kind though.

Reply:
Yes, case insensitive seems probably the right thing? I don't feel strongly about it though.